packages/core/admin/admin/src/components/GuidedTour/index.js

```diff
@@ -15,10 +16,14 @@ const GuidedTour = ({ children }) => {
   );
 
   const setCurrentStep = step => {
-    const isStepAlreadyDone = get(guidedTourState, step);
+    // if step is null it is intentional, we need to dispatch it
+    if (step !== null) {
+      const isStepAlreadyDone = get(guidedTourState, step);
+      const isStepToShow = arePreviousStepsDone(step, guidedTourState);
```

Comment:
This naming seems inconsistent, `isStepToShow` lets us think that it (and only it) controls the step visibility. But the visibility is also depends on `isStepAlreadyDone` and `isSkipped` WDYT?